src/modules/card.js

```diff
@@ -0,0 +1,8 @@
+// // eslint-disable-next-line import/no-cycle
+import getModalContainer from './card-content.js';
+
+const displayCard = (imgURL, name, itemID, summary) => {
+  getModalContainer(imgURL, name, summary, itemID);
+};
+
```

Comment:
You can remove this file. I tried running it using npm and it raised an error. Your code works just fine without it